TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
```

Comment:
I think we discussed this in person a bit. No particular reason, though I don't like the tradeoff of making the variable an optional in order to initialize it in viewDidLoad unless necessary. Do you think it would be best to do all variable init in viewDidLoad?